src/Config/ObjectModel/RuntimeConfig.cs

```diff
@@ -466,9 +462,9 @@ public static bool IsHotReloadable()
     /// 2. Multiple create operation is enabled in the runtime config.
     /// 
     /// </summary>
-    public bool IsMultipleCreateOperationSupportedAndEnabled()
+    public bool IsMultipleCreateOperationEnabled()
     {
-        return (RELATIONAL_DBS_SUPPORTING_MULTIPLE_CREATE.Contains(DataSource.DatabaseType)) &&
+        return Enum.GetNames(typeof(MultipleCreateSupportingDatabaseType)).Any(x => x.Equals(DatabaseType.MSSQL.ToString(), StringComparison.OrdinalIgnoreCase)) &&
```

Comment:
Can you elaborate on what is happening here? You're checking that `MultipleCreateSupportingDatabaseType` has `DatabaseType.MSSQL` defined.. which it does. This will always return true, even if the runtime config were set for DW/My/PG/Cosmos?

Reply:
Thanks for spotting this...Miss on my end.

Updated to `DataSource.DatabaseType.ToString()`